src/styles/main.scss

```
@@ -1,3 +1,94 @@
+@import 'variables';
+@import 'card';
+@import 'stars';
+
+:root {
+  --blue-color: #00ACDC;
+}
+
+*,
+*::before,
+*::after {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+	font-family: Roboto, sans-serif;
+  font-size: 12px;
+  color: #000;
+  background-color: #fff;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+.header {
+  width: 100vw;
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #fff;
+}
+
+.header__logo {
+  display: flex;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  flex-direction: row;
+}
+
+.nav__item {
+  list-style-type: none;
+  height: 60px;
+  margin: auto 10px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
```

```suggestion
.nav__item:first-child,
.nav__item:last-child {
  margin-right: 0;
}
```
FYI you may join selectors with same properties